packages/config-loader/src/sources/RemoteConfigSource.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ResponseError } from '@backstage/errors';
+import { HumanDuration, JsonObject } from '@backstage/types';
+import isEqual from 'lodash/isEqual';
+import fetch from 'node-fetch';
+import yaml from 'yaml';
+import { ConfigTransformer, createConfigTransformer } from './transform';
+import {
+  AsyncConfigSourceIterator,
+  ConfigSource,
+  SubstitutionFunc,
+  ReadConfigDataOptions,
+} from './types';
+
+const DEFAULT_RELOAD_INTERVAL = { seconds: 60 };
+
+function durationToMs(duration: HumanDuration): number {
+  const {
+    years = 0,
+    months = 0,
+    weeks = 0,
+    days = 0,
+    hours = 0,
+    minutes = 0,
+    seconds = 0,
+    milliseconds = 0,
+  } = duration;
+
+  const totalDays = years * 365 + months * 30 + weeks * 7 + days;
+  const totalHours = totalDays * 24 + hours;
+  const totalMinutes = totalHours * 60 + minutes;
+  const totalSeconds = totalMinutes * 60 + seconds;
+  const totalMilliseconds = totalSeconds * 1000 + milliseconds;
+
+  return totalMilliseconds;
+}
+
+/**
+ * Options for {@link RemoteConfigSource.create}.
+ *
+ * @public
+ */
+export interface RemoteConfigSourceOptions {
+  /**
+   * The URL to load the config from.
+   */
+  url: string;
+
+  /**
+   * The interval in seconds to reload the config from the remote URL.
+   *
+   * Set to Infinity to disable reloading.
```

Comment:
```suggestion
   * Set to undefined (default) to disable reloading.
```

Reply:
It's actually how it works still, default is 1 minute. Updated with a better example tho